README.md

```diff
@@ -1,96 +1,290 @@
+-   [Back & Front End Engineering](#back--front-end-engineering)
+    -   [Configuration](#configuration)
+    -   [Pipeline](#pipeline)
 
+# Welcome to Climate Change AI!
+
+Our project aims to raise **awareness and conceptual understanding of climate change** by bringing the future closer.
+
+We will do this by developing an interactive website to depict accurate and personalized outcomes of climate change using cutting-edge techniques from artificial intelligence and climate modeling.
+
+Bringing together researchers from different areas, we aim to design an educational tool that will produce **accurate** and **vivid** renderings of the **future outcomes of climate change** as they are likely to affect individuals.
+
+# TECHNOLOGIES
+
+An overview of the different technologies we are using for modeling, computing and rendering the outcomes of Climate Change on the **back end Flask server**.
+
+## NVIDIA CUDA®
+
+CUDA® is a parallel computing platform and programming model developed by NVIDIA for general computing on graphical processing units (GPUs). With CUDA, developers are able to dramatically speed up computing applications by harnessing the power of GPUs.
+
+In GPU-accelerated applications, the sequential part of the workload runs on the CPU – which is optimized for single-threaded performance – while the compute intensive portion of the application runs on thousands of GPU cores in parallel. When using CUDA, developers program in popular languages such as C, C++, Fortran, Python and MATLAB and express parallelism through extensions in the form of a few basic keywords.
+
+The [CUDA Toolkit](https://developer.nvidia.com/cuda-toolkit) from NVIDIA provides everything you need to develop GPU-accelerated applications. The CUDA Toolkit includes GPU-accelerated libraries, a compiler, development tools and the CUDA runtime.
+
+[Nvidia Cuda](https://developer.nvidia.com/cuda-zone)
+
+![Nvidia Cuda](https://devblogs.nvidia.com/wp-content/uploads/2018/09/image8-300x296.png)
+
+## SPADE
+
+We propose spatially-adaptive normalization, a simple but effective layer for synthesizing photorealistic images given an input semantic layout. Previous methods directly feed the semantic layout as input to the deep network, which is then processed through stacks of convolution, normalization, and nonlinearity layers. We show that this is suboptimal as the normalization layers tend to ``wash away’’ semantic information. To address the issue, we propose using the input layout for modulating the activations in normalization layers through a spatially-adaptive, learned transformation. Experiments on several challenging datasets demonstrate the advantage of the proposed method over existing approaches, regarding both visual fidelity and alignment with input layouts. Finally, our model allows user control over both semantic and style. Code is available at [this https URL](https://github.com/NVlabs/SPADE) .
+
+[Spade](https://nvlabs.github.io/SPADE/)
```

Comment:
Checkout the rendered version, this doesn't render properly: https://github.com/cc-ai/floods-backend/blob/9c165aa6b065767d93c27a62ef8776f5ceeb2e9b/README.md